stories/002.CreatingAComponent.stories.mdx

```diff
@@ -0,0 +1,185 @@
+   ```bash
+   mkdir packages/nightingale-hello
+   ```
+2. Create a folder for your source code `src/` and one for your tests `test/`:
+   ```bash
+   mkdir packages/nightingale-hello/src
+   mkdir packages/nightingale-hello/tests
+   ```
+3. Create the `package.json` for the new component. Here is an example of its content:
+
+   ```json
+   {
+     "name": "nightingale-hello",
+     "version": "4.0.0",
+     "description": "Basic usage",
+     "files": ["dist", "src"],
+     "main": "dist/index.js",
+     "module": "dist/index.js",
+     "type": "module",
+     "types": "dist/index.d.ts",
+     "scripts": {
+       "build": "rollup --config ../../rollup.config.mjs"
+     },
+     "publishConfig": {
+       "access": "public"
+     },
+     "keywords": ["nightingale", "webcomponents", "customelements"],
+     "repository": {
+       "type": "git",
+       "url": "https://github.com/ebi-webcomponents/nightingale.git"
+     },
+     "bugs": {
+       "url": "https://github.com/ebi-webcomponents/nightingale/issues"
+     },
+     "homepage": "https://ebi-webcomponents.github.io/nightingale/",
+     "dependencies": {
+       "@nightingale-elements/nightingale-new-core": "^4.0.0",
+       "d3": "7.8.2"
+     }
+   }
+   ```
+
+4. All nightingale components are written in TypeScript. We need a file `tsconfig.json` with the following content for TypeScript compilation.
+
+   ```json
+   {
+     "extends": "../../tsconfig.json",
+     "compilerOptions": {
+       "outDir": "./dist"
+     },
+     "include": ["./src"]
+   }
+   ```
+
+5. Create the entrypoint for webpack. The current setup of the nightingale project looks for `src/index.ts`:
+
+  ```typescript
+  import NightingaleHello from "./nightingale-hello";
+
+  export default NightingaleHello;
+  ```
+
+6. Create the file where the code of your custom element will be. In our example `src/nightingale-hello.ts`.
+
+   1. To start we can create a Hello world document with this code:
+
+      ```typescript
+      import { LitElement, html } from "lit";
+      import { customElement } from "lit/decorators.js";
+
+      export
+      @customElement("nightingale-hello")
+      class NightingaleHello extends LitElement {
+        @property({ type: String, attribute: "name" })
+        name = "";
+
+        render() {
+          return html`<p>Hello ${this.name}!</p>`;
+        }
+      }
+
+      export default NightingaleHello;
+      ```
+
+      Here are some notes of this code:
+
+      - The library used to build the web components in Nightingale is [Lit](https://lit.dev/docs/).
+      - The `@customElement` decorator is used for defining the custom element following the web component standard of defining and registering custom elements.
+      - The component rendering is defined by the method `render()`.
+
+   2. The above example inherits from [LitElement](https://lit.dev/docs/api/LitElement/), while there is a base element for any Nightingale component named `NightingaleElement`.
```

Comment:
I think is worth splitting this in 2 points:
2. Use `NightingaleElement` instead of `LitElement`...
3. Use of mixins, maybe adding one at a time.